extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -2621,11 +2556,6 @@ public String getCurrentZoneName() {
         return stateMap.get("CurrentZoneName");
     }
 
-    public String getCurrentURIFormatted() {
```

Comment:
This is one of the few methods of the handler which are not used. If you want, I can restore it with a call to the new update method.